src/components/helpers/utils.ts

```diff
@@ -10,3 +10,6 @@ export const GBdate = (date) => {
   return d.length > 1 ? `${d[2]}/${d[1]}/${d[0]}` : date;
 }
 
+export const checkErrorMsgs = (errorMsgs=[],id="",name="") => {
+  return (errorMsgs.find((element) => (element.message.fieldId === id) || (element.message.fieldId.startsWith(name))));
```

Comment:
You can simplify this methods like 

```
export const checkErrorMsgs = (errorMsgs=[],name="") => {
  return (errorMsgs.find((element) => (element.message.fieldId.toLowerCase().includes(name))));
}
```

Reply:
Basically Form Group is being used by multiple components , so one of them is sending "id" and some other is sending "name" , so in order to satisfy either the condition , we have this logic.